aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -11,6 +11,7 @@ module aptos_framework::code {
     use aptos_std::copyable_any::Any;
     use std::option::Option;
     use std::string;
+    use aptos_framework::object::{Self, Object};
```

Comment:
Can you sort this alphabetically

Reply:
Is this convention? Looking at other modules and they all have `Self` in the beginning of the imports. [Reference here](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/aptos_governance.move#L27-L33)